### DISCUSSION TITLE
[Feature] Allow configuration of user account recovery time

### DISCUSSION THREAD

**Discussion by ddshd** [immich-app/immich#3543] [Feature Request]
### The feature

Currently user account can be recovered after 7 days, add some method to allow immediate deletion or allow configuration of the time the account can be recovered until 

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by jrasm91** [comment]
I think there is a duplicate request for immediately deleting the user, but I like the idea of making it configurable. We could put that in the admin settings >users page possibly.

**Comment by samholton** [comment]
Related https://github.com/immich-app/immich/discussions/1616

Putting this next on my list

**Reply by samholton** [reply]
First round of changes in https://github.com/immich-app/immich/pull/7663

Force immediate delete up next
